### DISCUSSION TITLE
How can I get the current git branch name from nushell?

### DISCUSSION THREAD

**Discussion by LifeWorks** [nushell/nushell#8802] [Q&A]
I want to alias some of my git commands similar to oh-my-zsh style alias.

**Comment by fdncred** [comment]
nushell doesn't have git integration but I personally use the nushell gstat plugin.

**Reply by cooperspencer** [reply]
How do I integrate that plugin in my prompt? I compiled and registered it, but how do I proceed?

**Reply by fdncred** [reply]
You can see how I did it by looking at the `oh-my.nu` prompt in the nu_scripts repo. You can see several prompts here https://github.com/nushell/nu_scripts/tree/main/modules/prompt

**Reply by cooperspencer** [reply]
thanks a lot

**Comment by amtoine** [comment]
otherwise, the current Git branch name can be retrieved with
```nushell
let branch = git branch --show-current
```
:yum:
